### PR TITLE
DOC: fix rst formating

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Fixing trivial formatting which makes the doc render wrong.

### CODE REVIEW

File: joblib/parallel.py [hunk 1]
```diff
@@ -144,13 +144,13 @@ class parallel_backend(object):
...
 
-    >> from ray.util.joblib import register_ray
-    >> register_ray()
-    >> with parallel_backend("ray"):
-    ..     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))
-    [-1, -2, -3, -4, -5]
+     >>> from ray.util.joblib import register_ray  # doctest: +SKIP
+     >>> register_ray()  # doctest: +SKIP
+     >>> with parallel_backend("ray"):  # doctest: +SKIP
+     ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))  # doctest: +SKIP
```
Comment (pierreglaser) [inline joblib/parallel.py:152]: To appease the linter, maybe deactivate warning on the long line:
```suggestion
     ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))  # doctest: +SKIP  # noqa
```

Comment (GaelVaroquaux) [reply joblib/parallel.py:152]: Thanks!!

Comment (pierreglaser) [reply joblib/parallel.py]: mmh, now, doctests are failing... https://github.com/PyCQA/pydocstyle/issues/234#issuecomment-294282208 suggest to put the `# noqa` one the line of the function definition.

File: joblib/parallel.py [hunk 2]
```diff
@@ -144,13 +144,13 @@ class parallel_backend(object):
...
 
-    >> from ray.util.joblib import register_ray
-    >> register_ray()
-    >> with parallel_backend("ray"):
-    ..     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))
-    [-1, -2, -3, -4, -5]
+     >>> from ray.util.joblib import register_ray  # doctest: +SKIP
+     >>> register_ray()  # doctest: +SKIP
+     >>> with parallel_backend("ray"):  # doctest: +SKIP
+     ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))  # doctest: +SKIP  # noqa
```
Comment (jeremiedbb) [inline joblib/parallel.py:152]: maybe it's the line too long here.

**Comment by GaelVaroquaux** [comment]
OK, I've found the right combination that makes both pytest and the linter happy.

The remainer failures are unrelated to this PR, and related to pypy3 not downloading right.

This is ready to merge

**Review by tomMoral — APPROVED** [review]
LGTM! thanks @GaelVaroquaux for taking the time to make the CI happy!

**Comment by tomMoral** [comment]
I opened a PR to fix the `pypy` download link : #1114 
Merging this one to get a nice doc!

**Pull request merged**